src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -231,22 +232,23 @@ def get_storage_devices_topology(self):
         return topology
 
     async def get_gpus(self):
+
         if self.GPU:
             return self.GPU
 
         not_available = {'available': False, 'vendor': None}
-        cp = await run(['lshw', '-numeric', '-C', 'display'], check=False)
-        if cp.returncode:
-            self.logger.error('Unable to retrieve GPU details: %s', cp.stderr.decode())
+
+        # we only support nvidia GPUs at the moment
+        cmd = ['lspci', '-d', NVIDIA_PCI_ID + VGA_CLASS_ID]
+        vga = await run(cmd, check=False)
+        if vga.returncode:
+            self.logger.error('Unable to retrieve GPU details: %s', vga.stderr.decode())
             return not_available
 
-        vendor = RE_GPU_VENDOR.findall(cp.stdout.decode())
-        if not vendor:
+        nvidia = [i for i in vga.stdout.decode().strip().split('\n') if 'nvidia' in i.lower()]
```

Comment:
How about just doing `'nvidia' in vga.stdout.decode().lower()` ?

Reply:
Yeah, that reads better. I've changed it to that.